threat_intel/util/http.py

```diff
@@ -273,10 +274,31 @@ def _wait_for_response(self, requests, to_json):
             A list of dicts if to_json, a list of grequest.response otherwise
         """
         all_responses = []
-
-        for request, response in zip(requests, grequests.map(requests)):
+        
+        retry = 0
+        max_retry = 10
+        responses = None
+        while retry <= max_retry and responses is None:
```

Comment:
you can write 'and not response', which is more pythonic. Then while responses is None, not None is True. When response is not None, not not None is not True, which is False.